README.md

```diff
@@ -48,6 +48,21 @@ pip install -r requirements.txt
  - DTD: https://www.robots.ox.ac.uk/~vgg/data/dtd/
  - CIFAR10: http://www.cs.toronto.edu/~kriz/cifar.html
 
+For a quick start with CelebAHQ, you might:
```

Comment:
I just wanted to facilitate people's life by copy-pasting the TLDR.

Reply:
Good idea for celebaHQ, it was a pain in the *** to make the first time. 

I would however add a section ## Quick download